agent/ui_endpoint.go

```diff
@@ -161,7 +164,7 @@ RPC:
 
 	// Generate the summary
 	// TODO (gateways) (freddy) Have Internal.ServiceDump return ServiceDump instead. Need to add bexpr filtering for type.
-	return summarizeServices(out.Nodes.ToServiceDump()), nil
+	return summarizeServices(out.Nodes.ToServiceDump(), s.agent.config), nil
```

Comment:
Should we use the `s.agent.AutoConfig` to get the configuration? 🤔

Reply:
Worth verifying with @mkeeler, but I think not. When building a DNSServer we still use `s.agent.config` to get the domain.